<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>1011</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="radarView">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="pieView">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="historyView">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>590</y>
     <width>831</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="correctNumber">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>correctRate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Radar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>correctness</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>900</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>corectness</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>910</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>corectness</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>530</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
